javers-core/src/test/groovy/org/javers/core/JaversDiffE2ETest.groovy

```diff
@@ -172,4 +172,36 @@ class JaversDiffE2ETest extends Specification {
         then:
         DiffAssert.assertThat(diff).hasChanges(0)
     }
+
+
+    def "should serialize the Diff object"() {
```

Comment:
zapomniałeś o klasach do zmian na kolekcjach - ContainerElementChange,
dopisałem Ci to do testu, który teraz failuje